test/test_items.rb

```diff
@@ -2,27 +2,35 @@
 load 'test/test_harness.rb'
 
 class TestItems < Minitest::Test
-  def setup
-    load('common.lic')
+  def teardown
+    @test.join if @test
```

Comment:
Is the idea with this to build up the tests then execute them all at once?

Reply:
Or what is happening here?

Reply:
It prevents the test from thinking it's done before the thread that the actual test is in has finished running. Otherwise it can just move on and say "All good, no failed assertions" even though the script is still running in another thread.